zh_cn/news/_posts/2014-11-13-ruby-1-9-3-p551-is-released.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: news_post
+title: "Ruby 1.9.3-p551 发布"
+author: "usa"
+translator: "cassiuschen"
+date: 2014-11-13 12:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布 Ruby 1.9.3-p551 发布了。
+
+本次发布包括了一个针对 REXML 拒绝服务攻击（DoS）漏洞的安全修复。
+
+下述日志中包含相似内容：
+[预发布](https://www.ruby-lang.org/en/news/2014/10/27/ruby-1-9-3-p550-is-released/) 中的 [漏洞修复](https://www.ruby-lang.org/en/news/2014/10/27/rexml-dos-cve-2014-8080/),
```

Comment:
> [预发布](https://www.ruby-lang.org/en/news/2014/10/27/ruby-1-9-3-p550-is-released/)
if no zh_cn available, please keep en version as is.

> 中的 [漏洞修复]

no spaces between Chinese characters.

> (https://www.ruby-lang.org/en/news/2014/10/27/rexml-dos-cve-2014-8080/),

Use Chinese punctuations.